src/vm/fieldmarshaler.h

```diff
@@ -69,6 +69,48 @@ class FieldMarshaler_Exception;
 class FieldMarshaler_Nullable;
 #endif // FEATURE_COMINTEROP
 
+enum ParseNativeTypeFlags
+{
+    ParseNativeTypeFlag_None = 0x00,
+    ParseNativeTypeFlag_IsAnsi = 0x01,
+
+#ifdef FEATURE_COMINTEROP
+    ParseNativeTypeFlag_IsWinRT = 0x02,
+#endif // FEATURE_COMINTEROP
+};
+
+inline ParseNativeTypeFlags operator|=(ParseNativeTypeFlags& lhs, ParseNativeTypeFlags rhs)
```

Comment:
This isn't the correct signature - `ParseNativeTypeFlags& operator|=(ParseNativeTypeFlags& lhs, ParseNativeTypeFlags rhs)`;